_modules/ext_module_qvm.py

```diff
@@ -883,7 +883,10 @@ def run_post(cmd, status, data):
 
         # Value matches; no need to update
         value_new = kwargs[key]
-        if value_current == value_new:
+        vals_are_none = value_new == 'none' and value_current is None
+        noneable_keys = ['netvm', 'dispvm_netvm', 'kernel', 'drive']
+        equiv_none = vals_are_none and key in noneable_keys
```

Comment:
Happy to do this differently. It was just the most readable way I could come up with under 80 chars / line.